src/08_example/04_tests.md

```diff
@@ -0,0 +1,53 @@
+# Testing Async Code
```

Comment:
Sounds pretty general, perhaps we can be more specific that we're testing our code?